src/collections/_documentation/guides/migration/index.md

```diff
@@ -0,0 +1,59 @@
+
+To make this migration as seamless as possible, Sentry has a migration workflow that allows you to migrate most of your account's settings and configurations including project settings, alert rules, enabled data scrubbers, team settings, saved search queries, integration platform metadata (Sentry 10), etc.
+
+One thing to note is that we won't be able to transfer over historical event data, but new incoming events will populate the projects in your new organization. You will also need to re-invite all your members and add them to the correct teams.
+
+## How to Migrate
+
+### 1. Sign up
+
+Before you begin the migration process, [sign up](https://sentry.io/signup/) for a new organization in Sentry.
+
+![Sign Up]({% asset guides/migration/01.png @path %})
+
+Once you complete the onboarding steps, grab your **org slug** from the browser URL or under `Settings > General Settings`. We'll need this info to know where to migrate your data to.
+
+![Org Slug]({% asset guides/migration/02.png @path %})
+
+> Note: You can modify your org slug in the org settings.
+
+### 2. Choose a plan
+
+Take a look at our [plans](https://sentry.io/pricing/) and select one that suits your needs based on the different features and capabilities. Notice that plan pricing varies based on the type of _Contract_ and _Monthly Event Volume_, so make sure you are familiar with your current/expected monthly usage (number of events per month).
+
+If you're expecting higher volumes or interested in our Enterprise capabilities (priority support and SLAs, single-tenant options, etc.), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
+
+### 3. Export your data
+
+Sentry open-source installs a [Command-Line](https://docs.sentry.io/server/cli/) script that allows you to perform various operations that are unachievable within the web UI. One of those is `export`, exporting your data into a transport JSON.
+
+Run the following command in your terminal to start the export script and redirect the output (_containing_ the transport JSON) to a file:
+
+    $ docker-compose run --rm web export > sentry_export.txt
+    
+> This invokes the sentry CLI `export` command on a new isolated instance of our Docker web container.
+
+### 4. Contact us
+
+To kick-off the migration process shoot us an email with the following details at <support@sentry.io>:
+
+- Email Subject: `Self-Hosted Migration Application`
+- Your **organization slug**
+- Your **selected plan**. If you're unsure, let us know. We'll be happy to help you make the right decision. Also, if you think you'll need an Enterprise plan we can connect you with the right person to discuss that plan.
+- Attach the **export output** containing the transport JSON.
+
+Alternatively, you can also reach out to us through [Switch to Hosted Sentry](https://sentry.io/from/self-hosted/)
```

Comment:
Maybe we should promote this over the e-mail as reading those steps make me cringe a bit (feels cumbersome, especially the way we ask people to share the export file with us).